specification/cognitiveservices/data-plane/Speech/SpeechToText/stable/v3.0/speechtotext.json

```diff
@@ -6151,6 +6154,12 @@
               }
             },
             "x-ms-error-response": true
+          },
+          "503": {
```

Comment:
Is there a reason this doesn't have the `x-ms-error-response` annotation?  Seems weird to treat a 503 as a success.

Reply:
The response type is the same as the 200/OK response and not an error type. We did not add this intentionally because of this.

Reply:
This is very strange.  I have never seen a `503` response that returns the same response body as a `200-OK`.  Are you sure that is correct?  If the service is "temporarily unavailable" (the meaning of the 503 status code), what created a request that _looks just like_ a response from the service?

Reply:
@bexxx Can you please explain how a 503 response can be a successful response?  That is highly unusual!

Reply:
The route in question is the health route. The One API response body knows three states: Healthy, Degraded and Unhealthy. If a service instance is degraded or unhealthy, we take it from the load balancer and if it doesn't become healthy, the instance gets restarted. For this we use the [health check feature of Azure App Services](https://docs.microsoft.com/en-us/azure/app-service/monitor-instances-health-check). This feature requires to return a 5xx in case the service is degraded or unhealthy. 

The challenge is, that the documented response type for the health check is different from the standard error response type. In this case the response body contains the health response type. This is the same response type, that also gets returned in case the service is healthy and we return 200.